2-mandatory/3-financial-times.js

```diff
@@ -14,7 +20,10 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  let arr = allArticleTitles;
+  let newArr = arr.sort((a, b) => a.length - b.length);
+  return newArr[0];
+  // TODO
 }
```

Comment:
Nice try and well done on using sort() function on an array. However, this function should sort articles by no of words instead of the total length of the article. A little further modification is needed.

Reply:
i am still trying to figure this one out.

Reply:
Hint: You can split a sentence into an array of words by applying split() function.